src/main/python/image_preprocessing/face_manager.py

```diff
@@ -61,14 +68,9 @@ def extract_face(self, image):
             obj:'OpenCV image': Any background and unnecessary components are removed and only
             the aligned face is returned.
             obj:'OpenCV image': A copy of the original image is returned.
-        Todo:
-            Return error if no face detected
-
         """
         rectangle = self.detect(image)
-        predictor = dlib.shape_predictor(self.shape_predictor_path)
-        face_aligner = FaceAligner(predictor, desiredFaceWidth=256)
-        face_aligned = face_aligner.align(image, image, rectangle)
+        face_aligned = self.face_aligner.align(image, image, rectangle)
         image_copy = image.copy()
         return face_aligned, image_copy
```

Comment:
Sorry, I know that this line was inserted a while ago and it isn't actually part of this pull request. Why return a copy of the image that was passed through? I think this is also maybe something that slows down the system, because I do not think that this image copy is used anywhere else. If it is, rather change it so that a copy of the image is made before this function call.